client/src/amplify-ui/cli-feature-flag/feature-flags.json

```diff
@@ -143,21 +143,105 @@
         "valueType": "Boolean",
         "versionAdded": "4.40.0",
         "deprecationDate": "May 1st 2021",
         "values": [
           {
             "value": "true",
-            "description": "Allows using newly migrated codegen package.",
+            "description": "Allows using newly migrated modelgen package.",
             "defaultNewProject": true,
             "defaultExistingProject": false
           },
           {
             "value": "false",
-            "description": "Allows using older codegen package",
+            "description": "Allows using older modelgen package",
+            "defaultNewProject": false,
+            "defaultExistingProject": true
+          }
+        ]
+      },
+      "useDocsGeneratorPlugin": {
+        "description": "Allows to use the migrated graphQL documents generator package at https://github.com/aws-amplify/amplify-codegen",
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.40.0",
+        "deprecationDate": "May 1st 2021",
+        "values": [
+          {
+            "value": "true",
+            "description": "Allows using newly migrated docsgen package.",
+            "defaultNewProject": true,
+            "defaultExistingProject": false
+          },
+          {
+            "value": "false",
+            "description": "Allows using older docsgen package",
+            "defaultNewProject": false,
+            "defaultExistingProject": true
+          }
+        ]
+      },
+      "useTypesGeneratorPlugin": {
+        "description": "Allows to use the migrated graphQL types generator package at https://github.com/aws-amplify/amplify-codegen",
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.40.0",
+        "deprecationDate": "May 1st 2021",
+        "values": [
+          {
+            "value": "true",
+            "description": "Allows using newly migrated typesgen package.",
+            "defaultNewProject": true,
+            "defaultExistingProject": false
+          },
+          {
+            "value": "false",
+            "description": "Allows using older typesgen package",
+            "defaultNewProject": false,
+            "defaultExistingProject": true
+          }
+        ]
+      },
+      "cleanGeneratedModelsDirectory": {
+        "description": "Clears the generated models folder before they are re-generated.",
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.40.0",
+        "deprecationDate": "May 1st 2021",
+        "values": [
+          {
+            "value": "true",
+            "description": "Recommended to be set true when there are no hand-written files in the modelgen output folder.",
+            "defaultNewProject": true,
+            "defaultExistingProject": false
+          },
+          {
+            "value": "false",
+            "description": "Does not remove pre-existing files from the modelgen output folder.",
```

Comment:
Why wouldn't this always be set? Presumably it's because removing a `@model` from the schema definition will not automatically remove the generated file. If so, is that something we should call out here?

Reply:
This feature is enabled by default for new projects, but the existing projects have to set this to `true` based on their use-case. Elaborated on this to call out the default behavior.